webpack.base.config.js

```diff
@@ -116,7 +130,8 @@ module.exports = {
 
   plugins: [
     new HtmlWebpackPlugin({
-      template: "src/index.html"
+      chunksSortMode: 'none',
```

Comment:
Why chunksSortMode: 'none' ?  Add comment in code...thanks

Reply:
It has been removed! This was because when I first integrated the wiki it wasn't working without this option, but now I just tested it and it works :_) - So i'm removing it